Makefile

```
@@ -41,6 +41,10 @@ setup-venv:
 diff-format:
 	$(IN_VENV) darker -r $(TARGET_BRANCH)
 
+format:
+	$(IN_VENV) black .
+	$(IN_VENV) isort .
```

```suggestion
	$(IN_VENV) isort .
	$(IN_VENV) black .
```

I feel black should have the last word?

---

Sure, good idea.  I did test running them repeatedly in different orders and they're currently not in conflict (i.e. -- after they've both been run, there are no changes when running either, so order shouldn't currently matter), but it does feel like black should go last.